apps/checkout-app/mocks/handlers/saleor.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  PrivateMetafieldsInferedQuery,
+  PrivateMetafieldsInferedQueryVariables,
+} from "@/checkout-app/graphql";
+import { appPrivateMetafields } from "../fixtures/saleor";
+import { prepareGraphqlMetafields, saleorApi } from "../utils";
+
+export const saleorHandlers = [
+  saleorApi.query<
+    PrivateMetafieldsInferedQuery,
+    PrivateMetafieldsInferedQueryVariables
+  >("PrivateMetafieldsInfered", (req, res, ctx) => {
+    const { keys } = req.variables;
+    console.log(keys);
```

Comment:
We probably shouldn't leave console logs like this?

Reply:
Whoops, good catch 😅